app/components/FilterableTable/Filters/EnumFilter.tsx

```diff
@@ -48,7 +48,9 @@ export default class EnumFilter<T> extends TableFilter {
               key={this.argName + '-' + kvp.display}
               value={kvp.value as any}
             >
-              {getUserFriendlyStatusLabel(kvp.display)}
+              {this.ignoreFriendlyDisplay
+                ? kvp.display
+                : getUserFriendlyStatusLabel(kvp.display)}
             </option>
```

Comment:
For better OOP and future reuse, instead of a boolean we could allow the user to supply the implementation of the transformation function - in this case with the identity function.

And since this only affects the behaviour of the EnumFilter we probably shouldn't modify the parent `TableFilter` class ("open for extension, closed for modification")

I'd do something like that 
```javascript
export interface IEnumFilterOptionSettings<T> extends ISearchOptionSettings {
  renderEnumValue?(label: T): string;
}
export default class EnumFilter<T> extends TableFilter {
  enumValues: T[];

  constructor(
    display,
    argName,
    enumValues: T[],
    settings?: IEnumFilterOptionSettings
  ) {
    super(display, argName, settings);
    this.renderEnumValue =
      settings.renderEnumValue ?? getUserFriendlyStatusLabel;
      
      ...
```
and then instantiate with the identity
```javascript
new EnumFilter<String>(
    'Facility Type',
    'facilityType',
    ['SFO', 'IF_a', 'IF_b', 'L_c'],
    {renderEnumValue: (label) => label, sortable: false}
  )
```